src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -635,6 +648,21 @@ public void PlaceOrganelle()
         if (ActiveActionName == null)
             return;
 
+        if (isMovingOrganelle)
+        {
+            isMovingOrganelle = false;
+            ActiveActionName = SelectedActionName;
+            GetMouseHex(out int q, out int r);
+            var organelleHere = editedMicrobeOrganelles.GetOrganelleAt(new Hex(q, r));
+            if (organelleHere != null)
+                return;
+
+            MoveOrganelle(selectedHex, new Hex(q, r));
+            organelleHere = editedMicrobeOrganelles.GetOrganelleAt(new Hex(q, r));
```

Comment:
This is a redundant lookup, move organelle should already set the MovedThisSession properly so this isn't needed.